react/examples.spec.jsx

```diff
@@ -63,7 +63,6 @@ testComponent('PercentageLine')
 testComponent('Progress')
 testComponent('ProgressionBanner')
 testComponent('QuotaAlert')
-testComponent('Radio')
```

Comment:
deprecated doesn't mean "unsupported" and "untested" no?

Reply:
on a fait passer une PR dernièrement qui supprime les tests sur les composants dépréciés. Pour simplifier je serai plutôt d'avis de continuer dans cette logique, car si je suis plutôt d'avis de dire que "déprécié" veut dire "n'est plus supporté"